### DISCUSSION TITLE
no photos tab or archive

### DISCUSSION THREAD

**Discussion by fergy1** [photoprism/photoprism#2781] [Q&A]
Hi all 
Newbie here. I have installed Photoprism  and am unable to find a way to archive/delete photos etc. After visiting the Photoprism site is indicates that  I should have a photos tab. I only have an Albums tab and no archive tap or photos tab. Any help would be appreciated.
I have ticked the delete box in settings.

**Comment by graciousgrey** [comment]
Photos has been renamed to Search. If you click the arrow down next to "Search" on the navigation you find the archive. Alternatively you can also use the browser toolbar to navigate to the archive (example.com/archive)

- https://docs.photoprism.app/user-guide/organize/archive/
- https://docs.photoprism.app/user-guide/organize/delete/

**Comment by fergy1** [comment]
Thanks heaps thought I was going mad
Cheers

Get Outlook for Android<https://aka.ms/ghei36>
________________________________
